readme.md

```diff
@@ -270,6 +270,24 @@ Each top-level component receives a `url` property with the following API:
 
 The second `as` parameter for `push` and `replace` is an optional _decoration_ of the URL. Useful if you configured custom routes on the server.
 
+##### With URL object
+The component `<Link>` can also receive an URL object and it will automatically format it to create the URL string.
+
+```jsx
+// pages/index.js
+import Link from 'next/link'
+export default () => (
+  <div>Click <Link href={{ pathname: 'about', query: { name: 'Zeit' }}}<a>here</a></Link> to read more</div>
+)
+```
+
+That will generate the URL string `/about?name=Zeit`, you can use the following parameters:
```

Comment:
I can link the Node.js documentation yeah, that's probably better.

Yes, all properties work, you can create a URL right now using them manually, I don't think it's a problem, the only bug I watched about that is that if you don't set a protocol and set `host` and/or `auth` the `isLocal` function return true even if it's not 🤔